### ISSUE TITLE
RemoteCactbot not loading on iOS

### ISSUE THREAD

**Issue by RadiantKakas** [quisquous/cactbot#5279]
## Description

On both Safari and Chrome in iOS (tested on both iPad Air (4th generation) running iOS 16.3 and iPhone 12 mini running iOS 16.2), RemoteCactbot just loads a blank page (it never shows the character selector). The console has 1 error, "SyntaxError: Invalid regular expression: invalid group specifier name - raidboss_data.bundle.js:93960". The same URL loads on both Android, Windows, and macOS browsers.

## Additional information

![image](https://user-images.githubusercontent.com/125339156/218873754-8678b48a-745a-4e8c-9f0c-bbab870b02de.png)

**Comment by RadiantKakas** [comment]
https://stackoverflow.com/questions/51568821/works-in-chrome-but-breaks-in-safari-invalid-regular-expression-invalid-group

It looks like its related to the previous line's regex

**Comment by quisquous** [comment]
Yeah, this is a known issue.  iOS (really WebKit) doesn't support regex lookbehind.  There's not really much that can be done about it.

**Comment by RadiantKakas** [comment]
Ahh shoot, that makes sense. Oh well!

**Comment by valarnin** [comment]
I commented this on the original PR (because I didn't see this issue, because I really should be asleep at 3am to be honest), so copy/pasting here as well.

> Doing some additional testing now that it's live, and It looks like it doesn't load _at all_ on iOS. But, I think that would fall under a new issue? Tried in both Safari and Chrome and even the initial page where you choose the character is just blank.

All iOS browsers use webkit behind the scenes, even Chrome or Firefox.

cactbot raidboss uses lookbehind in regexes.

Chromium (which is what OverlayPlugin uses via CEF) has supported this since 2017. Firefox has supported this since 2020.

webkit (and therefore iOS in general, regardless of what browser frontend you use) will finally support this feature in its next release I assume, since it's listed as supported in the current Technology Preview release. No clue when that release will actually happen.

https://caniuse.com/js-regexp-lookbehind
